### ISSUE TITLE
Video Embeds

### ISSUE THREAD

**Issue by chrisdaaz** [nulib/homicide-in-chicago#6]
For the gallery section, learn how to use JavaScript to embed  the video element from another website into this website.

**Comment by chrisdaaz** [comment]
hey @echoe320 -- taking you off this. i wrote this for myself but forgot to claim it, but i think we'll put the video in a play that produces embed code.

**Comment by chrisdaaz** [comment]
waiting on the embed codes from the video repository.. on hold for now..

**Comment by chrisdaaz** [comment]
video embeds have been added
